packages/stateful/proposal-module-adapter/adapters/CwdProposalSingle/components/ui/ProposalVoteTally.tsx

```diff
@@ -50,19 +50,22 @@ export const ProposalVoteTally = ({
     ? turnoutAbstainPercent
     : totalAbstainPercent
 
+  // Single choice proposals must have a threshold.
+  const prop_threshold = threshold!
```

Comment:
can we make this camel case? and maybe just for a cleaner diff, rename the prop like `threshold: _threshold` at the top, and then define this here as `const threshold = _threshold!`. just my stylistic preference